docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md

```diff
@@ -98,52 +103,55 @@ LongWindowOption
 目前只支持长窗口`LONG_WINDOWS`的优化选项。
 
 #### 长窗口优化
-##### 长窗口优化选项格式
 ```sql
 LongWindowDefinitions
 						::= 'LongWindowDefinition (, LongWindowDefinition)*'
 
 LongWindowDefinition
-						::= 'WindowName[:BucketSize]'
+						::= WindowName':'[BucketSize]
 
 WindowName
 						::= string_literal
 
-BucketSize（可选，默认为）
+BucketSize
 						::= int_literal | interval_literal
 
-interval_literal ::= int_literal 's'|'m'|'h'|'d'（分别代表秒、分、时、天）
+interval_literal ::= int_literal 's'|'m'|'h'|'d'
 ```
-其中`BucketSize`为性能优化选项，会以`BucketSize`为粒度，对表中数据进行预聚合，默认为`1d`。
+其中`BucketSize`为用于性能优化的可选项，OpenMLDB会根据`BucketSize`设置的粒度对表中数据进行预聚合，默认为`1d`。
 
-示例如下：
-```sqlite
-DEPLOY demo_deploy OPTIONS(long_windows="w1:1d") SELECT col0, sum(col1) OVER w1 FROM t1
-    WINDOW w1 AS (PARTITION BY col0 ORDER BY col2 ROWS_RANGE BETWEEN 5d PRECEDING AND CURRENT ROW);
--- SUCCEED: deploy successfully
-```
 
 ##### 限制条件
 
 目前长窗口优化有以下几点限制：
-- 仅支持`SelectStmt`只涉及到一个物理表的情况，即不支持包含`join`或`union`的`SelectStmt`
+- `SelectStmt`仅支持只涉及一个物理表的情况，即不支持包含`join`或`union`的`SelectStmt`。
 
-- 支持的聚合运算仅限：`sum`, `avg`, `count`, `min`, `max`, `count_where`, `min_where`, `max_where`, `sum_where`, `avg_where`
+- 支持的聚合运算仅限：`sum`, `avg`, `count`, `min`, `max`, `count_where`, `min_where`, `max_where`, `sum_where`, `avg_where`。
 
-- 执行`deploy`命令的时候不允许表中有数据
+- 执行`deploy`命令的时候不允许表中有数据。
 
-- 对于 `count_where`, `min_where`, `max_where`, `sum_where`, `avg_where` 带 where 条件的运算，有额外限制：
+- 对于带 where 条件的运算，如 `count_where`, `min_where`, `max_where`, `sum_where`, `avg_where` ，有额外限制：
 
   1. 主表必须是内存表 (`storage_mode = 'Memory'`)
 
-  2. Deploy 的 `BucketSize` 类型需要是 range 类型，即支持 `long_windows='w1:1d'`, 不支持 `long_windows='w1:100'`
+  2. `BucketSize` 类型应为范围类型，即取值应为`interval_literal`类，比如，`long_windows='w1:1d'`是支持的, 不支持 `long_windows='w1:100'`。
 
-  3. where 条件必须是 `<column ref> op <const value> 或者 <const value> op <column ref>`的格式
+  3. where 条件必须是 `<column ref> op <const value> 或者 <const value> op <column ref>`的格式。
 
      - 支持的 where op: `>, <, >=, <=, =, !=`
 
      - where 关联的列 `<column ref>`，数据类型不能是 date 或者 timestamp
 
+#### Example
+```sql
+DEPLOY demo_deploy OPTIONS(long_windows="w1:1d") SELECT c1, sum(c2) OVER w1 FROM demo_table1
+    WINDOW w1 AS (PARTITION BY c1 ORDER BY c2 ROWS_RANGE BETWEEN 5d PRECEDING AND CURRENT ROW);
+
+I0908 08:06:15.512660  1091 zk_client.cc:58] node watcher with event type 3, state 3

```

Comment:
There're stdout logs, will removed later, it's better to delete these.

Reply:
logs in both examples have been removed